tasks/uk/census/wrapper.py

```diff
@@ -271,11 +353,11 @@ def requires(self):
         deps = {
             'source_geom_columns': OutputAreaColumns(),
             'source_geom': OutputAreas(),
-            'source_data_columns': CensusColumns(suffix='oa'),
+            'source_data_columns': CensusColumns(),
```

Comment:
Could you explain me the meaning of the change from using those two suffixes?

Reply:
I was using one column name for each geography level, but in the rest of DO we use the same column name for all the related geometries. From a DB schema point of view, this creates one entry in the `OBS_Column` table that relates to each `OBS_Table`